google/cloud/security/scanner/scanner.py

```diff
@@ -200,42 +200,65 @@ def _flatten_violations(violations, flattening_scheme):
     """
 
     # TODO: Make this nicer
-    LOGGER.info('Writing violations to csv...')
     for violation in violations:
         if flattening_scheme == 'policy_violations':
             for member in violation.members:
+                violation_data = {}
+                violation_data['role'] = violation.role
+                violation_data['member'] = '{}:{}'.format(member.type,
```

Comment:
nit: I believe we're trying to use string substitution, so to be consistent...

`%s:%s' % (member.type, member.name)`

If possible fix throughout